tracer/test/snapshots/Iast.SqlInjection.AspNetMvc5.IastEnabled.verified.txt

```diff
@@ -0,0 +1,78 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet.request,
+    Resource: GET /iast/sqlquery,
+    Service: sample,
+    Type: web,
+    Tags: {
+      env: integration_tests,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.route: {controller}/{action}/{id},
+      http.status_code: 200,
+      http.url: http://localhost/Iast/SqlQuery?query=SELECT%20Surname%20from%20Persons%20where%20name%20=%20%27Vicent%27,
+      http.useragent: Mistake Not...,
+      language: dotnet,
+      runtime-id: Guid_1,
+      span.kind: server,
+      _dd.iast.enabled: 1,
+      _dd.iast.json:
+{
+  "vulnerabilities": [
+    {
+      "type": "SQL_INJECTION",
+      
+      "evidence": {
+        "value": "SELECT Surname from Persons where name = 'Vicent'",
+        "valueParts": [
+          {
+            "value": "SELECT Surname from Persons where name = 'Vicent'",
+            "source": 0
```

Comment:
Shouldn't we highlight the injectable `'Vincent'` part here or is that not implemented yet?

Reply:
Right now, we are injecting in our tests the whole sql sentence. That's why the whole sentence is shown. When we have propagation, we should probably update these tests in order to make them more realistic. I have added a Jira task to make sure that we don't forget to do that. https://datadoghq.atlassian.net/browse/APPSEC-7683